prog/weaver/main.go

```diff
@@ -508,7 +512,7 @@ func parseIPAMSeed(ipamSeed string) []mesh.PeerName {
 	return peerNames
 }
 
-func listenAndServeHTTP(httpAddr string, muxRouter *mux.Router) {
+func listenAndServeHTTP(httpAddr string) {
```

Comment:
I've actually just fixed this on master - 03afb54c - so this commit should be removed.